yup/src/yup.ts

```diff
@@ -17,12 +17,17 @@ const parseErrorSchema = (
     }
 
     if (validateAllFieldCriteria) {
+      const types = previous[error.path!].types;
+      const messages = types && types[error.type!];
+
       previous[error.path!] = appendErrors(
         error.path!,
         validateAllFieldCriteria,
         previous,
         error.type!,
-        error.message,
+        // eslint-disable-next-line
+        // @ts-expect-error
```

Comment:
![Screenshot 2021-04-14 at 17 09 57](https://user-images.githubusercontent.com/7545547/114738429-407be680-9d48-11eb-9151-32f62f1b2e43.png)

Reply:
I think you can do a cast here.

Reply:
Updated, you can review again